src/case-ui-toolkit.module.ts

```diff
@@ -71,6 +72,7 @@ import { PipesModule, CaseReferencePipe, MarkdownComponent, MarkdownModule, Base
       CaseViewComponent,
       CaseViewerComponent,
       SearchFiltersComponent,
+      CreateCaseFiltersComponent,
```

Comment:
Historically we used to import every newly migrated component into CaseUIToolkitModule. This approach was there because initially this repo was a very small library of components: see /src/components and we used to need all of them so grouping made sense. It has grown in size considerably since then.

I realised we are importing a lot of unnecessary components as part of it in man web whenever we import it like AlertModule.

We should have a module per component approach and import only the necessary modules. Also future clients should not need to import a one module with all components in it when they will need just one or two.

Also see here: https://stackoverflow.com/questions/46434830/should-i-create-a-module-per-component-in-angular-4-app.

I will follow that from now on as well. Feel free to ask questions.

Reply:
I've taken this out now.